test-engineering/contract-tests/README.md

```diff
@@ -39,26 +40,73 @@ MTS.
 
 For more details see the partner [README][partner_readme]
 
-### client
-
-The `client` directory contains a Python-based test framework for the
-contract tests. The HTTP client used in the framework requests tiles from the
-MTS and performs checks against the responses. The framework implements response
-models for the MTS API.
-
-For more details see the client [README][client_readme]
-
 ### volumes
 
 The `volumes` directory contains subdirectories which will be mounted as
 volumes into the Docker containers used in the contract test suite:
 
-- the `volumes/partner` directory contains a YML file which defines every
-response that the API returns keyed by form-factor and then os-family
-- the `volumes/contile` directory contains files that need to be provided to a
-MTS Docker container such as a partner settings file
 - the `volumes/client` directory contains a YML file which defines every test
 scenario that the contract test suite will run
+- the `volumes/contile` directory contains files that need to be provided to a
+MTS Docker container such as a partner settings file
+- the `volumes/partner` directory contains a YML file which defines every
+response that the API returns keyed by form-factor and then os-family
+
+
+## Local Execution
+
+To run the contract tests locally, execute the following from the repository root:
+
+**Build Contile Docker Image**
```

Comment:
thank you for documenting this 🙂 now my newbie docker self can stop editing docker-compose and accidentally commit it now 🙃